packages/button/src/Button/Button.tsx

```diff
@@ -119,8 +133,8 @@ const Root = <T extends TgphElement>({
         rounded={roundedMap[size]}
         data-tgph-button
         data-tgph-button-layout={layout}
-        data-tgph-button-active={active}
-        disabled={disabled}
+        data-tgph-button-state={state}
```

Comment:
Made a more general `state` data attribute so we can target `active`, `disabled`, `loading`, etc via css data attributes